### PR TITLE
disruptor, style edits

### PR DESCRIPTION
**Pull request by MattDodsonEnglish**
imperative headings for tasks, shorter sentences, more list formatting, and so on.

### CODE REVIEW

File: src/data/markdown/docs/40 xk6-disruptor/03 Examples/01 Inject HTTP faults into Pod.md [hunk 1]
```diff
@@ -36,11 +36,14 @@ export default function(data) {
 
 <Blockquote mod="note">
 
- The test uses the `delay` endpoint which return after the requested delay. It requests a `0.1s` (`100ms`) delay to ensure the baseline scenario (see scenarios below) has meaningful statistics for the request duration. If we were simply calling a locally deployed http server (for example `nginx`), the response time would exhibit a large variation between a few microseconds to a few milliseconds. Having `100ms` as baseline response time has proved to offer more consistent results.
+The test uses the `delay` endpoint which, returns after the requested delay.
```
Comment (MattDodsonEnglish) [inline src/data/markdown/docs/40 xk6-disruptor/03 Examples/01 Inject HTTP faults into Pod.md:39]:
```suggestion
The test uses the `delay` endpoint, which returns after the requested delay.
```

**Comment by MattDodsonEnglish** [comment]
@pablochacin  These edits are quite minor, and uncontroversial. But I wanted to flag you for a quick review just for move visibility.

File: src/data/markdown/docs/40 xk6-disruptor/01 Get started/02 Requirements.md [hunk 2]
```diff
@@ -4,25 +4,31 @@ excerpt: 'Requirements for using xk6-disruptor in your test scripts'
...
+  Kubernetes clusters can be configured to restrict the download of images from public repositories. You need to ensure this image is available in the cluster where the system under test is running. Additionally, the xk6-disruptor-agent must run with network access privileges. Kubernetes clusters [can be configured to restrict the privileges of containers](https://kubernetes.io/docs/concepts/security/pod-security-admission/).
 If you find an error similar to the following when using the xk6-disruptor, contact your cluster administrator and request the necessary privileges.
 
-> ERROR\[0000\] error creating PodDisruptor: pods "nginx" is forbidden: violates PodSecurity "baseline:latest": non-default capabilities (container "xk6-agent" must not include "NET_ADMIN", "NET_RAW" in securityContext.capabilities.add)
+  ```txt
+  ERROR\[0000\] error creating PodDisruptor: pods "nginx" is forbidden: violates PodSecurity "baseline:latest": non-default capabilities (container "xk6-agent" must not include "NET_ADMIN", "NET_RAW" in securityContext.capabilities.add)
+  ```
 
 
 You also need to ensure your test application is accessible from the machine where the tests run.
```
Comment (pablochacin) [inline src/data/markdown/docs/40 xk6-disruptor/01 Get started/02 Requirements.md:33]:
```suggestion
- Ensure your test application is accessible from the machine where the tests run.
```

File: src/data/markdown/docs/40 xk6-disruptor/03 Examples/01 Inject HTTP faults into Pod.md [hunk 3]
```diff
@@ -307,27 +315,35 @@ export const options = {
 
 ## Test setup
 
-The tests requires the deployment of the `httpbin` application. The application must also be accessible using an external IP available in the `SVC_IP` environment variable.
+To run these tests, you need to deploy the `httpbin` application.
```
Comment (pablochacin) [inline src/data/markdown/docs/40 xk6-disruptor/03 Examples/01 Inject HTTP faults into Pod.md:318]:
```suggestion
To run this test, you need to deploy the `httpbin` application.
```

**Review by pablochacin — CHANGES_REQUESTED** [review]
I like the changes. The texts is more fluid and terse. Only some minor changes.

**Comment by MattDodsonEnglish** [comment]
@pablochacin , do you mind checking one last edit in commit [de76dff](https://github.com/grafana/k6-docs/pull/950/commits/de76dff9eea1a0e0e86d6faaba0d15b2a2cefa76).
 
- I bolded the titles of the requirements to make them easier scan.
- I made the final item a noun phrase, to keep the list parallelism.
- Moved the Kubernetes version info to the very top, since it's essential info. I also try to be cautious of using admonitions  because of the [banner blindness](https://en.wikipedia.org/wiki/Banner_blindness) phenomenon.

I also created a merge conflict somehow :facepalm: ―something that happened when I made the edit with the github UI?―but I'll figure that out later.

File: src/data/markdown/docs/40 xk6-disruptor/01 Get started/02 Requirements.md [hunk 4]
```diff
@@ -3,26 +3,28 @@ title: 'Requirements'
...
 
- xk6-disruptor needs to interact with the Kubernetes cluster on which the application under test is running.
- To do so, you must have the credentials to access the cluster in a [kubeconfig](https://kubernetes.io/docs/tasks/access-application-cluster/configure-access-multiple-clusters/) file.
- Ensure that this file is pointed to by the `KUBECONFIG` environment variable or that it is located at the default location, `$HOME/.kube/config`.
 
-<Blockquote mod="note">
+To use xk6-disruptor in a k6 test script, you need the following:
 
-xk6-disruptor requires Kubernetes version 1.23 or higher
+- **A bundled k6 extension that includes the disruptor.**
```
Comment (pablochacin) [inline src/data/markdown/docs/40 xk6-disruptor/01 Get started/02 Requirements.md:12]: @MattDodsonEnglish I find the term "a boundle k6 extension" confusing. Is that the way we are referring now to the custom k6 binaries with bundled extensions?

I think the original phrase was something like "a custom k6 binary that bundles the xk6-disruptor extension".

File: src/data/markdown/docs/40 xk6-disruptor/01 Get started/02 Requirements.md [hunk 5]
```diff
@@ -3,26 +3,28 @@ title: 'Requirements'
 excerpt: 'Requirements for using xk6-disruptor in your test scripts'
 ---
 
-The xk6-disruptor is a k6 extension. 
-To use it in a k6 test script, you need to bundle a k6 extension that includes the disruptor.
-Refer to the [Installation](/javascript-api/xk6-disruptor/get-started/installation) section  for instructions on how to get this custom build.
+At this moment,
```
Comment (pablochacin) [inline src/data/markdown/docs/40 xk6-disruptor/01 Get started/02 Requirements.md:6]: No at this moment. It is an extension. And it is intended for systems running kubernetes. And it requires kubernetes 1.23. I prefer the original phrasing.

File: src/data/markdown/docs/40 xk6-disruptor/01 Get started/02 Requirements.md [hunk 6]
```diff
@@ -3,26 +3,28 @@ title: 'Requirements'
...
 
-> ERROR\[0000\] error creating PodDisruptor: pods "nginx" is forbidden: violates PodSecurity "baseline:latest": non-default capabilities (container "xk6-agent" must not include "NET_ADMIN", "NET_RAW" in securityContext.capabilities.add)
-
+  ```txt
+  ERROR\[0000\] error creating PodDisruptor: pods "nginx" is forbidden: violates PodSecurity "baseline:latest": non-default capabilities (container "xk6-agent" must not include "NET_ADMIN", "NET_RAW" in securityContext.capabilities.add)
+  ```
 
-You also need to ensure your test application is accessible from the machine where the tests run.
-Refer to [Expose your application](/javascript-api/xk6-disruptor/get-started/expose-your-application) section for instructions on how to make your application available from outside the Kubernetes cluster..
+- **A test application that is accessible from the machine where the tests run.**
```
Comment (pablochacin) [inline src/data/markdown/docs/40 xk6-disruptor/01 Get started/02 Requirements.md:29]: I prefer the original phrasing.

**Review by pablochacin — COMMENTED** [review]
@MattDodsonEnglish In general, I prefer the original phrasing. These changes are introducing some subtle changes in the meaning that I that I don't agree with. I understand you want to streamlined the text, but I think it is important to preserve relevant details. Sometimes less is not more but less.

File: src/data/markdown/docs/40 xk6-disruptor/01 Get started/02 Requirements.md [hunk 7]
```diff
@@ -3,26 +3,28 @@ title: 'Requirements'
...
+ Ensure that this file is pointed to by the `KUBECONFIG` environment variable or that it is located at the default location, `$HOME/.kube/config`.
 
+- **Access to the [grafana/xk6-disruptor-agent](https://github.com/grafana/xk6-disruptor/pkgs/container/xk6-disruptor-agent) image used by the [disruptor agent](/javascript-api/xk6-disruptor/explanations/how-xk6-disruptor-works/).**
 
-`xk6-disruptor` requires the [grafana/xk6-disruptor-agent](https://github.com/grafana/xk6-disruptor/pkgs/container/xk6-disruptor-agent) image for injecting the [disruptor agent](/javascript-api/xk6-disruptor/explanations/how-xk6-disruptor-works) into the disruption targets. Kubernetes clusters can be configured to restrict the download of images from public repositories. You need to ensure this image is available in the cluster where the application under test is running. Additionally, the xk6-disruptor-agent must run with network access privileges. Kubernetes clusters [can be configured to restrict the privileges of containers](https://kubernetes.io/docs/concepts/security/pod-security-admission/).
```
Comment (pablochacin) [inline src/data/markdown/docs/40 xk6-disruptor/01 Get started/02 Requirements.md:21]: I think you are removing relevant details here. Why is this image needed? I prefer the original phrasing.

**Comment by pablochacin** [comment]
Closing, as discussed with @MattDodsonEnglish, mostly due to changes in the wording are introducing subtle changes in the meaning.

**Pull request closed without merging**